website/content/docs/agent-and-proxy/agent/caching/index.mdx

```diff
@@ -7,7 +7,12 @@ description: |-
   newly created tokens.
 ---
 
-# Vault agent caching
+# Vault Agent caching
+
+~> Note: Unlike Vault Agent, [Vault Proxy](/vault/docs/agent-and-proxy/proxy) supports
```

Comment:
If we think this note will be useful in other places in the future, I would recommend making it into a partial so it's easier to maintain the language going forward.

Reply:
We're planning on deprecating the API proxy aspect of Agent in the future, so (and perhaps this is optimistic), we'll hopefully be able to remove this when we do. To put it another way, though, we won't be adding any new functionality or pages to Agent's API proxy functionality